Source/QuartzCore additions/CGPathAdditions.m

```diff
@@ -21,7 +21,7 @@ CGFloat fixInfinity(CGFloat inputFloat){
     return inputFloat;
 }
 
-CGPoint *fixPointsInfinity(CGPathElement *element){
+CGPoint *fixPointsInfinity(const CGPathElement *element){
```

Comment:
There's a warning about passing parameter to fixPointsInfinity in applier and applyPathTranslation which has const element parameter:
`void applier (void *info, const CGPathElement *element)`
`void applyPathTranslation (void *info, const CGPathElement *element)`